x-pack/plugins/secops/public/components/page/network/kpi_network/translations.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const NETWORK_EVENTS = i18n.translate('xpack.secops.kpiNetwork.source.networkEvents', {
+  defaultMessage: 'Network Events',
+});
+
+export const UNIQUE_ID = i18n.translate('xpack.secops.kpiNetwork.source.uniquiId', {
+  defaultMessage: 'Unique Flow ID',
+});
+
+export const ACTIVE_AGENTS = i18n.translate('xpack.secops.kpiNetwork.source.activeAgents', {
+  defaultMessage: 'Active Agents',
+});
```

Comment:
We have a few bad examples right now in our code base for `i18n` for keys. Sorry about that...I kind of started the bad habits.

This great guide was referred to us for making our key names more to up to par:
https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types

Guessing here but I think for these keys they should end with `Title` like:

```ts
xpack.secops.kpiNetwork.source.networkEventsTitle
```